doc/user/content/manage/user-management/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Role-based access control"
```

Comment:
Ah, something I failed to mention before: we should add the `{{< alpha />}}` tag in this page, to highlight that RBAC is disabled by default. This would maybe allow us to then remove the individual warnings added in #19293?